New Bank/NewBank/newbank/server/NewBank.java

```diff
@@ -83,11 +81,15 @@ public synchronized String processRequest(CustomerID customer, String request) {
 				String to = input[3];
 				return moveFunds(customer,amount,from,to);
 			}
+			if (input[0].equals("LOGOFF")){
+				return "LOGOFF";
+			}
```

Comment:
Moved the LOGOFF into the processRequest(CustomerID, String) instead as it wouldn't need to be handled while the user is logged out.